src/main/java/porcelli/me/git/integration/githook/push/GitHook.java

```diff
@@ -25,13 +26,14 @@
 import org.eclipse.jgit.transport.SshTransport;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+
+import com.jcraft.jsch.Session;
```

Comment:
please remove unused code.

Reply:
com.jcraft.jsch.Session is used in lines 126, 128, removing it will cause compilation errors ..

Reply:
ohh I see what happened, you formatted the code. please revert the changes of this file and it should be ok

Reply:
Done